app/Http/Requests/CategoryUpdateRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CategoryUpdateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => ['required', 'string', 'unique:categories,name']
```

Comment:
Does update work here? It should throw error name exists. You have to ignore it for this category witch is updating. https://laravel.com/docs/8.x/validation#rule-unique read `Forcing A Unique Rule To Ignore A Given ID:` in docs part